elements/lisk-chain/src/state_store/account_store.ts

```diff
@@ -11,8 +11,10 @@
  *
  * Removal or modification of this copyright notice is prohibited.
  */
-
-import { cloneDeep, isEqual, pick, uniq, uniqBy } from 'lodash';
+// tslint:disable-next-line no-require-imports
+import cloneDeep = require('lodash.clonedeep');
+// tslint:disable-next-line no-require-imports
+import isEqual = require('lodash.isequal');
```

Comment:
it should not use `require`

Reply:
Since we are not importing whole `lodash` library, we need to use `require` here.